runtime/queries/hurl/textobjects.scm

```diff
@@ -0,0 +1,5 @@
+(comment) @comment.inside
+
+(comment)+ @comment.around
+
+(entry (_) @entry.inner) @function.around
```

Comment:
Ah okay.. understood your comment.
Nevertheless I would like to suggest to use `@function.around`. Because users want to use e.g. m, a, f to select a whole call (what a function is) and pipe to hurl.
In the grammar it is called a `entry`, because hurl call this a entry: https://hurl.dev/docs/grammar.html#entry

Reply:
Ah I see, then the line might make more sense as `(entry (_) @function.inside) @function.around`